src/components/cards/TeamsCard.vue

```diff
@@ -6,31 +6,43 @@
     </v-card-title>
     <v-card-text>
       <div class="teams ma-1">
+        <v-btn icon ripple @click="dialog = true" v-if="teams.length === 0">
```

Comment:
This button is too small. Please make it bigger like the `add` one in https://v15.vuetifyjs.com/en/components/buttons#floating for example (but still grey)

Reply:
![github pr](https://user-images.githubusercontent.com/22354811/66913503-0b88b980-f015-11e9-9237-8e16a47d39da.PNG)

Like this?

Reply:
Yes!